### DISCUSSION TITLE
1分間隔の繰り返しを制限するかどうか（意見募集中）

### DISCUSSION THREAD

**Discussion by tetsuya-ki** [tetsuya-ki/discord-reminderbot#33] [Ideas]
現状、とくに制限していないため、1分間隔、かつ、無期限でリマインドを設定できる。
そのため、このBotによる荒らし行為ができてしまう。

1分間隔は、期限設定あり、かつ、少なめの回数(5回までとか？)のみ可能、みたいな仕様変更は必要だろうか。。。
常識的に考えて、荒らし行為に使えてしまうのは問題だよねぇ。。。
（休日にeveryoneにメンションするとかも可能なので、そういうのはいいのか？という気もする。深く考えてはいけないかも）

**Comment by m2me** [comment]
初めまして。
いつも運用しているDiscordサーバーにて、reminderbotを便利に活用させていただいております。

1分間隔の繰り返しを制限するかどうかについての意見となりますが、「制限する」かつ「期限設定、または回数制限有り」がいいのではないかと思います。（1日に1回、6回までなど）

仰る通り、荒し対策もそうですが、サーバー自体に負荷がかかるのではないかなと思ったためです。

**Comment by tetsuya-ki** [comment]
最近たまにあるので制限することにしました。分単位の場合は5回まで。
https://github.com/tetsuya-ki/discord-reminderbot/issues/78
